### ISSUE TITLE
After update and somes issues - printer don't run.

### ISSUE THREAD

**Issue by digitalarche** [bigtreetech/BIQU-Hurakan#1]
Then and after a meticulous installation for the cables, the tightenings, the leveling even manual and finally the park Bed&Mesh and the Z-Offset in order to have the perfect configuration I tried to make the configuration of the resonance with your accelerometer [By the way I wanted to point out that the PDF on your official site is not up to date compared to the file in your github and precisely the information on the SHAPER_CALIBRATE is in that of the github!]

Anyway, I install as specified in your DOC and I run the SHAPER_CALIBRATE AXIS=X command and I get an error message that Nympy is missing here is the error message: ["SHAPER_CALIBRATE" AXIS=X Failed to import `numpy` module, make sure it was installed via `~/klippy-env/bin/pip install` (refer to docs/Measuring_Resonances.md for more details)]

Having seen I think all the videos only available at the moment on the Hurakan I'm trying to see if anyone has had the same problem and just a youtuber who says he has had sames problems and you will revsolved for him.

Indeed, I had the OS and the Moonraker asking for an update and I had on the Klipper.Py an "invalid" information

So I upgraded the OS that went well and then I upgraded Moonraker and that's when everything went wrong it didn't want to boot at all it remained frozen on the Starting Boot.

I had backed up what was in the SD before anything so when I checked the dates of the files I see that two files have really changed so I put the old files back in the SDCard (again with a backup of it in the state) BUT nothing changes. The changes were made in the printer and nothing responds anymore.

I can ABSOLUTELY not use my machine anymore so do you have a factory reset at least but I don't dare (like your reset.txt) does it work for this one I'm not sure because it's not Marlin and I didn't find anything anywhere nor on your github.

How do I get my machine back to work? Effectively I open a support ticket on your website as well but I wanted 
to contact Bigtreetech as well here and leave this information if somone else will be the same issues. 

Thanks by advance on your really help. 
Franck,

**Comment by Geo-Warrior** [comment]
Hi. It's interesting to see this post. I'll be keeping an eye on it myself as I'm running into somewhat similar issues. I do hope that BTT starts to assist more with this printer as it's a really nice one with some quirks that are hard to address if, like me, you're unfamiliar with Linux and Klipper in general. 
Here is a link to the newest OS build... https://github.com/bigtreetech/CB1/releases
The notes look like it addresses some key issues so, good starting point. Specifically the third file(specifies Hurakan). I had my first failure with the printer due to the nozzle/fan shroud/microprobe assembly scraping into and dislodging my print from the plate. Since that happened, my microprobe won't deploy or retract the pin leading to it always having a triggered status. I don't know if it's related to the firmware update or not but still wanted to put it out there.

**Comment by digitalarche** [comment]
You're totally right It is far from easy to follow them, especially without any basic explanation of how the machine works and/or warning us not to do any updates for the moment or to be sure to have all the components installed in the machine. Because if I hadn't had this problem with Numpy, I'm not sure I would have launched into a systematic update? because as we often know, we have to let a few storms pass before upgrading anything. 

But since everything seemed to be working properly at the time and especially since it is plugged directly into the repos of the products in question and even worse that BigTreek has not blocked the fact of making updates, then when something does not work we try the update. I missed everything because the printer does not work at all and I do not know the manipulations as for the BX or others to perhaps put a system in factory mode. Having no idea nor documentation of the relationship between the SD/OS card and what is actually in the printer? do we have access to see this or at least correct/modify it if so I don't know how? 

Then finally and according to your POST indeed the OS and the img have just been updated but what to do with it I think I will try with BelenaEtcher a SDCARD but without much hope knowing that it is at the update of Moonraker that everything went wrong. Anyway I'm waiting to hear from them, none to date and I'm trying things without going too far to at least see if it can restart. Thanks again for your feedback, it's always interesting to know that we are not the only one with problems.

**Comment by Csiszerix** [comment]
I've had somewhat the same problem, while trying to install wireguard-client on my Hurakan. 
It installs it correctly and I can use it UNLESS I reboot the system. Had to reflash the SD card in order to get the printer to work. 
Can anyone recreate the same issue?
